extensions/hibernate-search-orm-elasticsearch/runtime/pom.xml

```diff
@@ -32,6 +32,17 @@
         <dependency>
             <groupId>org.hibernate.search</groupId>
             <artifactId>hibernate-search-mapper-orm</artifactId>
+            <exclusions>
+                <!-- We use Jakarta Persistence artifacts instead -->
+                <exclusion>
```

Comment:
Will you switch upstream soon as well?

Reply:
Not as long as ORM uses the `javax.*` artifacts. And I'm not sure it would be a good idea to do the switch before a major (e.g. Hibernate Search 7), considering the sneaky effects this can have on application dependencies (duplicate classes), even if the content of the JARs is essentially the same.

I only added this exclusion here because that's what we must do in Quarkus, apparently. I don't know exactly why it wasn't a problem in Hibernate Search before; I assume that in 6.0 we used to only have a transitive dependency to JPA through ORM, and now in 6.1 we have a direct dependency.

In any case the Hibernate Search artifacts for ORM 6 will, obviously, use Jakarta artifacts. That's probably enough?